lnrpc/routerrpc/router_server.go

```diff
@@ -178,22 +185,31 @@ func (s *Server) RegisterWithRootServer(grpcServer *grpc.Server) error {
 // PaymentRequest, then an error will be returned. Otherwise, the payment
 // pre-image, along with the final route will be returned.
 func (s *Server) SendPayment(ctx context.Context,
-	req *PaymentRequest) (*PaymentResponse, error) {
+	req *SendPaymentRequest) (*PaymentResponse, error) {
 
 	payment, err := s.cfg.RouterBackend.extractIntentFromSendRequest(req)
 	if err != nil {
 		return nil, err
 	}
 
-	preImage, _, err := s.cfg.Router.SendPayment(payment)
+	err = s.cfg.Router.SendPaymentAsync(payment)
 	if err != nil {
+		log.Debugf("SendPayment async result for hash %x: %v",
```

Comment:
`Errorf`?

Reply:
Fixed using different log level for expected and unexpected errors.